.sequelizerc

```diff
@@ -1,10 +1,13 @@
 require("@babel/register");
 
+const dotenv = require("dotenv");
+dotenv.config();
+
 const path = require('path');
 
 module.exports = {
-  'config': path.resolve('src/config', 'database.js'),
-  'models-path': path.resolve('src/models'),
-  'seeders-path': path.resolve('src/seeders'),
-  'migrations-path': path.resolve('src/migrations')
+  'config': path.resolve('api/database', 'database.js'),
+  'models-path': path.resolve('api/models'),
+  'seeders-path': path.resolve('./seeders'),
+  'migrations-path': path.resolve('./migrations')
```

Comment:
might be worth moving `seeders` and `migrations` inside the `api/database` folder

Reply:
Yeah, that makes sense to me